mapreduce/tests/test_e2e.py

```diff
@@ -0,0 +1,28 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import AgentCheck
+from datadog_checks.dev.utils import get_metadata_metrics, get_service_checks
+
+from .common import ELAPSED_TIME_BUCKET_METRICS, assert_metrics_covered, setup_mapreduce
+
+
+def test_e2e(dd_agent_check, instance):
+    # trigger a job but wait for it to be in a running state before running check
+    assert setup_mapreduce()
+
+    aggregator = dd_agent_check(instance, rate=True)
+
+    for metric in ELAPSED_TIME_BUCKET_METRICS:
+        # at_least=0 because sometimes the job is already done, and we don't get the metrics
```

Comment:
We could skip asserting the metrics altogether in this e2e test, it is verified in the integration test below.

Is there a failure this assertion would catch? Trying to understand in which scenario asserting metrics will add value, aka catch a failure.

Reply:
That's not true. 

[Here](https://github.com/DataDog/integrations-core/pull/16929/files#diff-228e4245602d6ce8556df194046054bf3fd297b4fb1b86103392e59a2f97ca43R19) was assert other metrics that can be collected during the tests. `ELAPSED_TIME_BUCKET_METRICS` can be flaky so if we do not assert them with `at_least=0`, we can't `assert_all_metrics_covered` because these metrics would not be asserted in some cases.

Reply:
Yes, I misread `ELAPSED_TIME_METRICS` for `ELAPSED_TIME_BUCKET_METRICS`, so you are right, these are not tested in integration tests.